src/style.css

```diff
@@ -1 +1,81 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.page__body {
+  margin-top: 0;
+}
+
+.search-bar--big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.search-bar__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-bar__input:focus {
+  outline: none;
+  font-weight: 900;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-image: url(images/Search.svg);
```

Comment:
Unfortunately, it doesn't work like that

You should write both: the gradient and image in one property, separated by coma (exactly as in my example)
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/83019fce-f330-454a-85fc-3b4888fa4bd7)

Also, check examples in links I've sent